coordinator/src/metrics.rs

```diff
@@ -49,6 +51,16 @@ lazy_static! {
         .u64_observable_gauge("node_balance_satoshi")
         .with_description("Node balance in satoshi")
         .init();
+
+    // position metrics
+    pub static ref POSITION_QUANTITY: ObservableGauge<f64> = METER
+        .f64_observable_gauge("position_quantity_contracts")
```

Comment:
nit: "position_quantity" or "position_contracts" should be enough. Having both is confusing